<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exporterUI</class>
 <widget class="QMainWindow" name="exporterUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Exporter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Asset:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="assetName_le"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Proyect:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="projects_cb"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Set:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="setName_le"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="createPivot_btn">
          <property name="text">
           <string>Create Pivot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectPivot_btn">
          <property name="text">
           <string>Select Pivot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="referenced_cbx">
          <property name="text">
           <string>Make Reference</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="savePosition_btn">
          <property name="text">
           <string>Save Position For Sel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="export_btn">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
